packages/blade/src/components/Radio/Radio.stories.tsx

```diff
@@ -27,13 +35,33 @@ const Page = (): React.ReactElement => {
         Radio & RadioGroup can be used in forms when a user needs to single value from several
         options.
       </Subtitle>
-      <Link withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
-        View in Figma
-      </Link>
+      <FigmaEmbed src={figmaURL} title="Radio Figma Designs" />
       <br />
       <br />
       <Title>Usage</Title>
-      <Highlight language="tsx">{`import { Radio, RadioGroup } from '@razorpay/blade/components' \nimport type { RadioProps, RadioGroupProps } from '@razorpay/blade/components'`}</Highlight>
+      <Sandbox showConsole>
+        {`
+          import { RadioGroup, Radio } from '@razorpay/blade/components';
+
+          function App(): JSX.Element {
+            return (
+              <RadioGroup
```

Comment:
Maybe follow the same content for Radio as Checkbox.

Reply:
done